erpnext/hr/report/employee_leave_balance/employee_leave_balance.py

```diff
@@ -51,7 +54,8 @@ def get_data(filters, leave_types):
 					filters.from_date, filters.to_date)
 
 				# opening balance
-				opening = get_total_allocated_leaves(employee.name, leave_type, filters.to_date)
```

Comment:
remove import get_total_allocated_leaves and also remove the method get_total_allocated_leaves from the leave_application.py file

Reply:
done